pypeman/channels.py

```diff
@@ -443,7 +451,7 @@ async def replay(self, msg_id):
 
         :return: The result of the processing.
         """
-        logger.info("try to replay %s", str(msg_id))
+        self.logger.info("try to replay %s", str(msg_id))
```

Comment:
shouldn't msg_id already be a simple object like an int or a string ?

Reply:
I'm not sure.... so str() is the more "robust" way to be sure it will be a simple object and then no socketLogger pickle problem will occur :)